packages/app-store/utils.ts

```diff
@@ -6,6 +6,8 @@ import type { TFunction } from "next-i18next";
 import { appStoreMetadata } from "@calcom/app-store/appStoreMetaData";
 import type { EventLocationType } from "@calcom/app-store/locations";
 import { defaultLocations } from "@calcom/app-store/locations";
+import getEnabledApps from "@calcom/lib/apps/getEnabledApps";
+import { prisma } from "@calcom/prisma";
```

Comment:
This module is imported client side(in `booking/[uid].tsx`) and thus it can't have prisma as a dependency in it.
![image](https://github.com/calcom/cal.com/assets/1780212/1bdca073-4552-4314-a555-e51a7060dea9)

We should move `getLocationGroupedOption` into it's own module and the prisma dependency would move there along with it.

Reply:
Nevermind: I think your approach would be much faster because you directly found out the parent teamId and then using just an **in** query.